flink/v1.14/build.gradle

```diff
@@ -17,18 +17,11 @@
  * under the License.
  */
 
-def flinkProjects = [
-  project(':iceberg-flink:iceberg-flink-1.14'),
-  project(':iceberg-flink:iceberg-flink-runtime-1.14')
-]
-
-configure(flinkProjects) {
-  project.ext {
-    flinkVersion = '1.14.0'
-  }
-}
+String flinkVersion = '1.14.0'
+String flinkMajorVersion = '1.14'
+String scalaVersion = System.getProperty("scalaVersion") != null ? System.getProperty("scalaVersion") : System.getProperty("defaultScalaVersion")
```

Comment:
We’ve recently also had work for supporting Scala 2.13 in Spark, so it might be wise to name `scalaVersion` to something more Flink related to keep the two separated. Though this isn’t a list of all known Scala versions for Flink so might not be a concern here.